src/ui/components/Shared/TracksList.react.js

```diff
@@ -155,10 +155,9 @@ export default class TracksList extends Component {
   }
 
   getTrackTiles() {
-    const self           = this;
-    const selected       = this.state.selected;
-    const tracks         = [...this.props.tracks];
-    const trackPlayingId = this.props.trackPlayingId;
+    const self = this;
```

Comment:
Can't this be dropped now? We use arrow functions, so `this` should be preserved